scipy/stats/tests/test_stats.py

```diff
@@ -7927,6 +7927,31 @@ def test_basic(self):
         assert_equal(F, 2.0)
         assert_allclose(p, 1 - np.sqrt(0.5), rtol=1e-14)
 
+    def test_unequal_var(self):
+        # toy samples with unequal variances
+        samples = [[-50.42, 40.31, -18.09, 35.58, -6.8],
+                   [23.44, 4.5, 15.1, 9.66, 27.71],
+                   [11.94, 11.1 , 9.87, 9.09, 3.33]]
```

Comment:
Please add a test with different number of observations in each sample (or just replace this one, since I don't think there's any special treatment for equal numbers of observations in each sample).